logger/log.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 var formatter = &textFormatter{
-	DisableTimestamp: true,
+	DisableTimestamp: false,
+	FullTimestamp:    true,
```

Comment:
I don't want timestamps in my logs. Can we have a config/flag for this please? I don't care if the default is on or off, as long as I can easily set it.